packages/kit/test/apps/basics/src/routes/__layout.svelte

```diff
@@ -9,6 +11,13 @@
 			}
 		};
 	}
+
+	if (browser) {
+		let h3 = document.createElement('h3');
```

Comment:
nit: this line and the next could both be `const`

Reply:
Thanks, I have changed it